distributed/bokeh/scheduler_html.py

```diff
@@ -163,6 +163,25 @@ def get(self):
         self.write(result)
 
 
+class PrometheusHandler(RequestHandler):
+    import prometheus_client # keep out of global namespace
```

Comment:
This will still get run at import time.    This is a problem for two reasons

1.  It adds about 11ms to our import time.   This is small, but these things add up somewhat quickly
2.  The full dashboard code will err if the user doesn't have prometheus_client installed

You'll probably want this in the `__init__` function